src/pageScanner/checks/paragraph-styled-as-header.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Check for elements with text content that is styled like a header.
+ *
+ * Axe-core has a built-in `p-as-heading` rule that checks for paragraphs
+ * that are styled like headings. That rule is less robust than this check,
+ * as it only checks for paragraphs with bold or italic text and gives back
+ * 'incomplete' or uncertain results for some of the test data that we want
+ * to flag for further checks.
+ *
+ * This check in this file takes into account the font size, length of the
+ * text, and considers paragraphs with large font size as headers as well.
+ *
+ * @param {Node} node The node to evaluate.
+ * @return {boolean} True if the node is styled like a header, false otherwise. Paragraphs with only bold or italic,
+ *                   are shorter than 50 characters, or are short with large font size are considered headers.
+ */
+
+import { fontSizeInPx } from '../helpers/helpers.js';
+
+export default {
+	id: 'paragraph_styled_as_header',
+	evaluate: ( node ) => {
+		const pixelSize = fontSizeInPx( node );
```

Comment:
I'm not sure what happens before this but is there any value in returning early here if node is empty?

```js
if ( ! node ) {
     return false;
}
```